homeassistant/components/motioneye/media_source.py

```diff
@@ -284,7 +284,13 @@ async def _build_media_path(
 
         sub_dirs: set[str] = set()
         parts = parsed_path.parts
-        for media in resp.get(KEY_MEDIA_LIST, []):
+        media_list = resp.get(KEY_MEDIA_LIST, [])
+
+        def get_media_sort_key(media: dict) -> str:
+            """Get media sort key."""
+            return media.get(KEY_MEDIA_SORT_ATTR, "")
```

Comment:
You can probably just use KEY_PATH that is already imported from the underlying client library, rather than needing to define a new const.

Reply:
Good point. Initially I wanted to go for timestamp thus made the constant for that. Then I noticed not all unit tests have, and realized sorting on something visible makes more sense.